.github/workflows/coverity.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Runs coverity-scan.sh every 24h on `master`
+name: Coverity Scan
+on:
+  schedule:
+    - cron: '0 1 * * *'
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        if: github.repository == 'netdata/netdata'
```

Comment:
We can actually put this on the job itself instead of on each individual stage. I'm fairly certain that that will result in the job not even getting queued instead of it running and skipping each step individually.

Reply:
Good point :)

Reply:
Done